library/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -30,6 +35,22 @@ public object Chucker {
             .setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
     }
 
+    public fun Context.createShortcut() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
+            getSystemService(ShortcutManager::class.java)?.let {
+                val shortcut = ShortcutInfo.Builder(this, "id1")
+                    .setShortLabel("Open chucker activity")
+                    .setLongLabel("Open chucker activity")
```

Comment:
These should come from resources.

Reply:
Also, please change it to `Open Chucker`. No need to specify that it is activity.

Reply:
Movd hardcode into resources.